test/fixtures/google-gax-packaging-test-app/src/index.ts

```diff
@@ -93,6 +102,8 @@ async function testShowcase() {
   await testPagedExpand(fallbackClient);
   await testWait(fallbackClient);
   await testPagedExpandAsync(fallbackClient);
+
+  await testExpand(restClient);
```

Comment:
Let's also add all others that work - `testEcho`, `testPagedExpand`, `testWait`, `testPagedExpandAsync`

Reply:
done.